packages/flutter_driver/lib/src/driver/profiling_summarizer.dart

```diff
@@ -5,8 +5,12 @@
 import 'percentile_utils.dart';
 import 'timeline.dart';
 
-/// The catrgory shared by all profiling related timeline events.
-const String kProfilingCategory = 'flutter::profiling';
```

Comment:
If we're no longer using 'flutter::profiling', how are we going to handle the code in https://github.com/flutter/engine/blob/f10f17257300b232d0b16c95c63a641a3e158207/shell/profiling/sampling_profiler.cc#L52 ?